drivers/flash/Kconfig

```diff
@@ -36,6 +36,17 @@ config FLASH_PAGE_LAYOUT
 	help
 	  Enables API for retrieving the layout of flash memory pages.
 
+config FLASH_EMULATOR
+	bool "Emulated flash driver"
+	depends on FLASH
+	default n
```

Comment:
`default y if BOARD_QEMU_X86 || BOARD_QEMU_CORTEX_M3`
the opposite `default n` is unnecessary as the 'n' is the automatic default value for any boolean entry in Kconfig.